src/BenchmarksClient/Workers/SignalRWorker.cs

```diff
@@ -11,6 +11,11 @@
 using Benchmarks.ClientJob;
 using Microsoft.AspNetCore.SignalR.Client;
 using Microsoft.AspNetCore.Http.Connections;
+
+// REVIEW: I copied this from https://github.com/aspnet/SignalR/commit/e7b84b753bf11739cbb2ef10fc0708b375ec816d#diff-6244ded017b0b3eff9bf3dbf575a9a72R55,
+// but the internal namespace is making me think this is wrong.
+using Microsoft.AspNetCore.SignalR.Internal.Protocol;
```

Comment:
Instead of `WithHubProtocol` you can use `AddMessagePackProtocol` and `AddJsonProtocol`. They call `WithHubProtocol` internally but you get to eliminate the internal namespace :)

Reply:
I was wondering if you were going to say that. The problem is `AddJsonProtocol` and `AddMessagePackProtocol` are defined in the Microsoft.AspNetCore.SignalR.Core project which the Client project doesn't take a dependency on. If you want to make this usable on the client, you'll probably want to move it to Common or something.

I see that you're currently using it in your Client tests, but I'm pretty sure that's only working because of the transitive dependency on Core through Tests.Utils.

Reply:
Good point. Will improve https://github.com/aspnet/SignalR/issues/1856